### PR TITLE
[SHARE-446][2.0.0] Redirect old urls to new urls

### PR DESCRIPTION
**Pull request by leb2dg**
* Redirect pks to new values
* Redirect curate to detail
* Fix entries error

### CODE REVIEW

File: app/routes/detail.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
 import Ember from 'ember';
 import { FRAGMENT_MAP, CONTROLLER_MAP } from '../utils/mappings';
+import ENV from '../config/environment';
+
+const NUM = 0xDEADBEEF;
+const MOD = 10000000000;
 
 export default Ember.Route.extend({
+    getEncodedPk(pk, type) {
+        const modelID = ENV.modelIDs[type] || 49;  // 49 is abstractcreativework
```
Comment (chrisseto) [inline app/routes/detail.js]: Could you change the default to `ENV.modelIDs. creativework`?

**Comment by leb2dg** [comment]
No... do you want curate to redirect to detail then?

**Comment by chrisseto** [comment]
Please 👍 , sorry for the confusion. 

> URLs like /curate/creativework/12345 should be redirected/rewritten to /creativework/ABCD-123-XYZ

**Comment by leb2dg** [comment]
@chrisseto, it redirects curate to detail now

**Pull request merged**